configs/projects/puppet-agent.rb

```diff
@@ -241,7 +241,20 @@
   end
   proj.component "ruby-shadow" unless platform.is_aix? || platform.is_windows?
   proj.component "ruby-augeas" unless platform.is_windows?
-  proj.component "openssl"
+
+  if ENV["USE_SYSTEM_OPENSSL"] == '1'
```

Comment:
@underscorgan @jsane I provided this guidance last time; I thought we'd added support for coercing those values but I realize now that was in the `packaging` repo, not Vanagon so it won't work here. I also expected that this value would _only_ ever be set in the positive, e.g. "USE_SYSTEM_OPENSSL=true bundle exec build", never "USE_SYSTEM_OPENSSL=false bundle exec build". YMMV, but my opinion is that using an extremely specific value is probably a bad idea (I can get into why but it boils down to Shell UX being _*dodgy*_), but here's a compromise (which uses an anonymous function, which I don't love but I could live with) that I think will work OK:

```ruby
def use_system_openssl?
  # explicitly catch the expected "don't use 
  return false if ENV["USE_SYSTEM_OPENSSL"] =~ /false/f/0/i
  !!ENV["USE_SYSTEM_OPENSSL"]
end

if use_system_openssl?
  # do the thing
end
```